exdb/forms.py

```diff
@@ -73,16 +79,13 @@ def clean(self):
         if ex_type.needs_verification and self.cleaned_data.get('start_datetime') < self.when:
             raise ValidationError(ex_type.name + " events cannot happen in the past")
 
-        if not ex_type.needs_verification and not self.cleaned_data.get('attendance'):
+        if not ex_type.needs_verification and (not self.cleaned_data.get(
+                'attendance') or self.cleaned_data.get('attendance') < 1):
             raise ValidationError(ex_type.name + " events must have an attendance")
 
         if not ex_type.needs_verification and not self.cleaned_data.get('audience'):
             raise ValidationError(ex_type.name + " events must have an audience")
 
-        if not ex_type.needs_verification and self.cleaned_data.get(
-                'attendance') and self.cleaned_data.get('attendance') < 1:
-            raise ValidationError(ex_type.name + " events must have an attendance greater than 0")
-
         if ex_type.needs_verification and self.cleaned_data.get('attendance'):
             raise ValidationError(ex_type.name + " events cannot have an attendance")
```

Comment:
I just figured we could tack it on since it should be fairly simple and you have changed some of them anyway. You make a good point however; I'll make an issue on Asana.